bootstrap/http/src/main/kotlin/com/studentcenter/weave/bootstrap/meeting/dto/MeetingTeamLocationResponse.kt

```diff
@@ -0,0 +1,30 @@
+package com.studentcenter.weave.bootstrap.meeting.dto
+
+import com.studentcenter.weave.domain.meeting.enums.Location
+
+data class MeetingTeamLocationResponse(
+    val locations: List<MeetingTeamLocationDto>
+) {
+
+    data class MeetingTeamLocationDto(
+        val name: String,
+        val displayName: String,
+        val isCapitalArea: Boolean
+    )
+
+    companion object {
+        private var instance: MeetingTeamLocationResponse? = null
+
+        fun getInstance(): MeetingTeamLocationResponse {
+
+            return instance ?: MeetingTeamLocationResponse(
+                Location.entries.map {
+                    MeetingTeamLocationDto(it.name, it.value, it.isCapitalArea)
+                }.also { instance }
+            )
+
+        }
```

Comment:
p4) 최근에 보다보니 코틀린에 lazyinit 말고 lazy 키워드를 이용해서 지연 초기화 및 val의 not null type이 가능하네요!
```suggestion
        private val instance by lazy {
            Location.entries.map {
                MeetingLocationsResponse.MeetingLocationDto(it.name, it.value, it.isCapitalArea)
            }
        }

        fun getInstance() = instance
```